src/aks-preview/azcli_aks_live_test/scripts/setup_venv.sh

```diff
@@ -3,6 +3,9 @@
 # check var
 # specify the version of python3, e.g. 3.6
 [[ -z "${PYTHON_VERSION}" ]] && (echo "PYTHON_VERSION is empty"; exit 1)
+[[ -z "${TOOL_STORAGE}" ]] && (echo "TOOL_STORAGE is empty"; exit 1)
```

Comment:
is this variable really vary in places?

Reply:
It will probably not change in the future, but it is changing in this PR

Reply:
Reverted.